kloppy/_providers/signality.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, Union
+
+from kloppy.domain import TrackingDataset
+from kloppy.infra.serializers.tracking.signality import (
+    SignalityDeserializer,
+    SignalityInputs,
+)
+from kloppy.io import FileLike, open_as_file
+
+
+def load(
```

Comment:
Can you add a docstring for this method? To me it is not obvious what the inputs (in particular `meta_data` and `venue_information`) are.